ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -28,7 +28,8 @@
           allow-whitelist-orgs-as-admins: true
           org-list:
             - ceph
-          trigger-phrase: ''
+          trigger-phrase: ''    
+          skip-build-phrase: 'qa tests .* skip build'
```

Comment:
That link says this feature is kind of broken :( but it does say that regexes are OK. Since it can scan either title or body, then we could do a very lenient phrase that allows a brief description (which you seem to want). For example `skip-build-phrase: '^jenkins do not test(.*)'`

That way you could do this in the PR body:

    jenkins do not test: qa changes